RecoPPS/ProtonReconstruction/plugins/CTPPSProtonProducer.cc

```diff
@@ -73,6 +81,8 @@ class CTPPSProtonProducer : public edm::stream::EDProducer<> {
   bool opticsValid_;
   edm::ESWatcher<CTPPSInterpolatedOpticsRcd> opticsWatcher_;
 
+  edm::ESGetToken<LHCInfoPerLS, LHCInfoPerLSRcd> lhcInfoPerLSToken_;
+  edm::ESGetToken<LHCInfoPerFill, LHCInfoPerFillRcd> lhcInfoPerFillToken_;
   edm::ESGetToken<LHCInfo, LHCInfoRcd> lhcInfoToken_;
```

Comment:
in this case these token are in the init list, so could be `const`

Reply:
Addressed in https://github.com/cms-sw/cmssw/pull/42515/commits/cd1fac4534f2fd682e4affde360e8f0344ec3cad